Exasol Geometry not displayed with new exasol drivers

#### System information: 
- Operating system (distribution) and version
- Windows 10
- DBeaver version
- 21.2.2
- Additional extensions
- None

#### Connection specification:
- Database name and version
- Exasol 7.0.11
- Driver name
- Exasol JDBC >= 6.2

#### Describe the problem you're observing:
Geometry data cannot be shown


#### Steps to reproduce, if exist:
Create table
`
create or replace table lines (name varchar(50), col1 geometry);

insert into lines values ('open', 'LINESTRING (11 1, 15 2, 15 10)'),
 ('closed', 'linestring (11 1, 15 2, 15 10, 13 10, 11 1)');

select * from lines;

`

![image](https://user-images.githubusercontent.com/5595511/136655874-55a08e0b-0a9f-4be9-8066-833662e1ffe9.png)

`org.jkiss.dbeaver.model.exec.DBCException: SQL Error: type error
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:55)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetDataReceiver.fetchRow(ResultSetDataReceiver.java:125)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.fetchQueryData(SQLQueryJob.java:721)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:544)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$0(SQLQueryJob.java:444)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:171)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:431)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:816)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:3440)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:118)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:171)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:116)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:4718)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: type error
	at com.exasol.jdbc.Column.type_error(Column.java:28)
	at com.exasol.jdbc.Column.getBytes(Column.java:133)
	at com.exasol.jdbc.EXAResultSet.getBytes(EXAResultSet.java:769)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCResultSetImpl.getBytes(JDBCResultSetImpl.java:411)
	at org.jkiss.dbeaver.data.gis.handlers.GISGeometryValueHandler.fetchBytes(GISGeometryValueHandler.java:227)
	at org.jkiss.dbeaver.data.gis.handlers.GISGeometryValueHandler.fetchColumnValue(GISGeometryValueHandler.java:83)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:49)
	... 14 more` 


#### Include any warning/errors/backtraces from the logs
<!-- Please, find the short guide how to find logs here: https://github.com/dbeaver/dbeaver/wiki/Log-files -->

--- a/plugins/org.jkiss.dbeaver.ext.exasol/src/org/jkiss/dbeaver/ext/exasol/model/ExasolGeometryValueHandler.java
+++ b/plugins/org.jkiss.dbeaver.ext.exasol/src/org/jkiss/dbeaver/ext/exasol/model/ExasolGeometryValueHandler.java
@@ -18,13 +18,23 @@ package org.jkiss.dbeaver.ext.exasol.model;
 
 import org.jkiss.code.NotNull;
 import org.jkiss.dbeaver.data.gis.handlers.GISGeometryValueHandler;
+import org.jkiss.dbeaver.model.exec.DBCException;
+import org.jkiss.dbeaver.model.exec.DBCSession;
 import org.jkiss.dbeaver.model.exec.jdbc.JDBCPreparedStatement;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
 import org.jkiss.dbeaver.model.exec.jdbc.JDBCSession;
+import org.jkiss.dbeaver.model.struct.DBSTypedObject;
 import org.locationtech.jts.geom.Geometry;
 
 import java.sql.SQLException;
 
 public class ExasolGeometryValueHandler extends GISGeometryValueHandler {
+	
+	@Override
+	protected Object fetchColumnValue(DBCSession session, JDBCResultSet resultSet, DBSTypedObject type, int index)
+			throws DBCException, SQLException {
+		return resultSet.getString(index);
+	}	
 
     protected void bindGeometryParameter(@NotNull JDBCSession session, @NotNull JDBCPreparedStatement statement, int paramIndex, @NotNull Geometry value) throws SQLException {
         statement.setString(paramIndex, value.toString()); // Just convert to string for Exasol (doesn't work with bytes)

